viostor/virtio_pci.c

```diff
@@ -197,12 +197,12 @@ static u16 vdev_get_msix_vector(void *context, int queue)
             } else {
                 vector = queue + 1;
             }
-        } else {
-            vector = VIRTIO_MSI_NO_VECTOR;
         }
     } else {
         /* on-device-config-change interrupt */
-        vector = VIRTIO_MSI_NO_VECTOR;
+        if (!adaptExt->msix_one_vector) {
```

Comment:
Thanks,
speaking about fallback  from MSI to ISR, it is not supported in Windows as far as I remember.
What we can try doing in the future is intercepting StorQueryResourceRequirements/StorFilterResourceReqirements calls and see if we can ask for allocating one vector more.